docs/mirroring/README.md

```diff
@@ -22,6 +22,15 @@ See: https://kubernetes.io/docs/contribute/style/content-guide/#dual-sourced-con
 
 <!--TODO: Generically identifying registry.k8s.io images in manifests / charts / addons.-->
 
+If you have a running cluster then our [community-images] [krew] plugin can
```

Comment:
having two links side by side is a little hard to distinguish, maybe re-wording to space them slightly

Reply:
or we might not need a krew link anyhow since community-images covers that?

Reply:
dropped the krew link